src/util-coredump-config.c

```diff
@@ -116,18 +122,23 @@ int32_t CoredumpLoadConfig (void)
             SCLogInfo ("Unexpected type for rlim_t");
             return 0;
         }
-        errno = 0;
         if (rlim_size == 8) {
-            max_dump = (rlim_t) strtoull (dump_size_config, NULL, 10);
+            ret = ByteExtractStringUint64(
+                    (uint64_t *)&max_dump, 10, strlen(dump_size_config), dump_size_config);
```

Comment:
This cast looks dangerous to me.

I would rather have `uint64_t max_dump64` and `uint32_t max_dump32` and then reconcile these